src/element.js

```diff
@@ -96,7 +96,7 @@ function addNormalRectPath(ctx, rect) {
 }
 
 export function shouldDrawCaption(rect, options) {
-  if (!options || !options.display) {
+  if (!options || (defined(options.display) && !options.display)) {
```

Comment:
Does it need to work with anything other than explicit `true`?
```suggestion
  if (!options || (options.display !== true) {
```

Reply:
OK! let me update it because I need to remove the import as well. I was thinking to test herlpers.merge with the dataset.captions and the chart.options.elements.treemap. What do you think ? (not here, of course)

Reply:
and rebase

Reply:
I'd avoid merging stuff. We should be using controller.options to access these, then the defaults are applied correctly.
Something like #121 (maybe I was too focused on scriptability and did not consider defaults at all)

Reply:
I fully agree! Another new thing to me. I think that for dividers, captions, labels (not strictly related to an element) should be move to options.

Reply:
> Does it need to work with anything other than explicit `true`?

It seems it doesn't work. Investigating....

Reply:
The `"options.display !== true"` is wrong because, if `display` is not set, returns `true` but the default of `captions.display` is `true`.

Reply:
Let me rollback your suggestion, rebase, and then let me check for another check even if I don't see many other ways.

Reply:
I'll have to take a look, I don't like having default handling all over the place. I'll check back later.

Meanwhile, if you can create a test catching the error, that would be good!

Reply:
> I'll have to take a look, I don't like having default handling all over the place. I'll check back later.

ok, in the meanwhile changed in `if (!options || options.display === false) {`. I think the best thing is what you wrote above, to go to options instead of element defaults.

> Meanwhile, if you can create a test catching the error, that would be good!

`captionAlign.js` test fixture has been changed in this PR in order to reproduce the issue. I will create another fixture specific for this issue.

Reply:
>  We should be using controller.options to access these, then the defaults are applied correctly.

I have created another branch to address this topic, following the other PR. Honestly, I don't see any fallback but maybe it depends on my lack of knowledge.